documentation/docs/components/components/card.md

```diff
@@ -50,6 +48,29 @@ The card container comprises of different elements.
 ## **Examples**
 
 <Section>
+<<<<<<< HEAD
```

Comment:
This indicates a git merge conflict. You need to resolve this one and other conflicts before pushing the changes.